### ISSUE TITLE
Sign Up Form Wireframe

### ISSUE THREAD

**Issue by RakanAlanazi** [billjcapps/CS5551Team3#1]

**Comment by RakanAlanazi** [comment]
I'm working on it.

**Comment by RakanAlanazi** [comment]
I uploaded a picture of the SignUp Form to the documentation folder. Please check it out and let me know what you think. @beauxq @billjcapps

**Comment by beauxq** [comment]
If we're using OAuth, we wouldn't have a sign up form. The user would only sign up with the OAuth provider.

**Comment by RakanAlanazi** [comment]
Right, then we only need login page that has a button ( Ex: Sign in with Google ) and logo for our web application or brief description about the application.
